src/ol/geom/geometry.js

```diff
@@ -130,8 +130,11 @@ ol.geom.Geometry.prototype.getClosestPoint = function(point, opt_closestPoint) {
 
 
 /**
+ * Returns true if this geometry includes the specified coordinate. If the
+ * coordinate is on the boundary of the geometry, returns false.
```

Comment:
This would be the common description of "contains".  However, it is not how this method behaves:
```js
var poly = new ol.geom.Polygon([[[0, 0], [1, 0], [1, 1], [0, 1], [0, 0]]]);
poly.containsCoordinate([0, 0]) // true
```